docs/docs/identity-providers.md

```diff
@@ -53,7 +53,7 @@ Enter a name for the application, select **Web app/API** as the **Application Ty
 
 ![Create application form](./microsoft/azure-create-application.png)
 
-Next you will need to create a key which will be used as the **Client Secret** in Pomeriunm's configuration settings. Click on **Keys** from the **Settings** menu.
+Next you will need to create a key which will be used as the **[Client Secret]** in Pomeriunm's configuration settings. Click on **Keys** from the **Settings** menu.
```

Comment:
s/Pomeriunm/Pomerium/

Reply:
Oh goodness. I suppose it's just a matter of time until I start typing pomeranian.